spec/models/point_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Point, type: :model do
+  it "has a constructor that can create instances" do
+    instance = described_class.new
+    expect(instance).to be_an_instance_of(described_class)
+  end
+
+  describe "it should be valid" do
+    it "with x- and y-value" do
+      point = described_class.new(x: 1.0, y: -1.5)
```

Comment:
Are negative coordinates useful in our context ?

Reply:
depends on how points are maintained for geojson. At the moment it's possible